scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1739,6 +1735,25 @@ def _initialize( self ):
         self.__verify_required_env_var('SCALYR_K8S_POD_NAMESPACE')
         self.__verify_required_env_var('SCALYR_K8S_NODE_NAME')
 
+        envars_to_log = [
+            'SCALYR_K8S_NODE_NAME',
+            'SCALYR_K8S_POD_NAME',
+            'SCALYR_K8S_POD_NAMESPACE',
+            'SCALYR_K8S_POD_UID',
+            'SCALYR_K8S_KUBELET_HOST_IP',
+            'SCALYR_K8S_CLUSTER_NAME',
+            'SCALYR_K8S_DISABLE_API_SERVER',
+            'SCALYR_K8S_CACHE_EXPIRY_SECS',
+            'SCALYR_K8S_CACHE_START_FUZZ_SECS',
+            'SCALYR_K8S_CACHE_EXPIRY_FUZZ_SECS',
+            'SCALYR_COMPRESSION_TYPE',
+            'SCALYR_ENABLE_PROFILING',
+            'SCALYR_PROFILE_DURATION_MINUTES',
+            'SCALYR_MAX_PROFILE_INTERVAL_MINUTES',
+        ]
+        for envar in envars_to_log:
+            self._logger.info("Environment variable %s : %s" % (envar, os.environ.get(envar, '<Not set>')))
```

Comment:
Looks like this is printing over and over
![image](https://user-images.githubusercontent.com/48775713/58140785-7a442100-7bf5-11e9-977d-cabd66405ea5.png)

Why is that so? This is in the `_initialize` method.  Shouldn't it be initialized once?

Reply:
I'm still wondering why `_initialize` keeps getting called once for every output of `agent_requests` status.  Hope that's not a bug!

Meanwhile, I've relocated the env variable printing elsewhere and verified it gets printed only once on startup.